auth/src/main/java/com/firebase/ui/auth/provider/GoogleProvider.java

```diff
@@ -46,14 +47,16 @@
     private static final int AUTO_MANAGE_ID = 1;
     private static final int RC_SIGN_IN = 20;
     private static final String ERROR_KEY = "error";
-    private static final String CLIENT_ID_KEY = "client_id_key";
     private GoogleApiClient mGoogleApiClient;
     private Activity mActivity;
     private IDPCallback mIDPCallback;
 
-    public GoogleProvider(FragmentActivity activity, IDPProviderParcel parcel, @Nullable String email) {
+    public GoogleProvider(
+            FragmentActivity activity,
+            @Nullable String email,
```

Comment:
Same question here about just passing IDPConfig (and elsewhere, I'll stop repeating myself)